packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/index.js

```diff
@@ -86,6 +86,7 @@ const FormModal = () => {
       const dynamicZoneTarget = query.get('dynamicZoneTarget');
       const forTarget = query.get('forTarget');
       const modalType = query.get('modalType');
+      const contentTypeKind = query.get('kind');
```

Comment:
You might need to add it to the `./utils/staticData.js` file as well. I know that the code works great without it but I think that it would be great for consistency.